app/models/server/models/Users.js

```diff
@@ -91,8 +91,11 @@ export class Users extends Base {
 		return this.update(_id, update);
 	}
 
-	findOnlineAgents() {
+	findOnlineAgents(agentId) {
 		const query = queryStatusAgentOnline();
```

Comment:
looks like `queryStatusAgentOnline` already expects an extra filter, so I'd ask to remove the added if and change for this:
```suggestion
		const query = queryStatusAgentOnline(agentId && { _id: agentId });
```

Reply:
Yeah, it makes sense.